app/controllers/api/functions.php

```diff
@@ -2222,13 +2222,22 @@
                 }
             }
 
+            $maxLogLength = 100000;
+            $logs = $executionResponse['logs'] ?? '';
+            $logsTruncated = false;
+
+            if (\is_string($logs) && \strlen($logs) > $maxLogLength) {
+                $logs = \substr($logs, 0, $maxLogLength) . "\n[WARNING] Logs truncated. The output exceeded {$maxLogLength} characters.";
```

Comment:
@samikshaaagarwal the issue was that the attribute could only save upto `100000` chars. If you truncate to `100000` and append the `warning..` string, it would again surpass the `100000` limit. So make sure that log length never crosses `100000` including the warning.